docs/reference/config-options.md

```diff
@@ -48,3 +49,10 @@ IMPORTANT: If you switch this back to `disabled-permanently` from
 `disabled-by-default`, LegalHold devices may still be active in teams
 that have created them while it was allowed.  This may change in the
 future.
+
+### Custom Search Visibility
+
+This sets the default setting for all teams, and can be overridden by
+customer support / backoffice. [Allowed
+values](https://github.com/wireapp/wire-server/blob/custom-search-visibility-limit-name-search/libs/galley-types/src/Galley/Types/Teams.hs#L382-L385):
+`disabled-by-default`, `enabled-by-default`.
```

Comment:
CustomSearchVisibilityType has clearer semantics - can you use clearer semantics also at the config level? If we add another setting with different logic, then "disabled" and "enabled" doesn't make sense anymore.

Reply:
This is only for enabling/disabling the feature, not to set any value which is why it follows the logic behind the other feature flags. I.e., you may turn this feature on/off using the backoffice but the actual value itself is to be set independently (a bit like SSO and Legahold with settings).